### DISCUSSION TITLE
Some disconnected wires on the beacons cannot be reconnected. These wires lead to components mounted inside the walls. Some of these walls are invulnerable.

### DISCUSSION THREAD

**Discussion by pavel4win** [FakeFishGames/Barotrauma#15229] [Ideas and suggestions]
Why does beacons even have impenetrable walls? It's unsportsmanlike and unrealistic.

Also, many players have difficulty restoring the wires leading to the docking port of a beacon. Player must to understand the structure of the docking port well in order to properly reconnect the disconnected wires. For me, this is not difficult, but 95% of players will not be able to do this.
I suggest fix it, so the wires leading to the docking port must be always properly connected.

**Comment by Regalis11** [comment]
> Why does beacons even have impenetrable walls? It's unsportsmanlike and unrealistic.

This has been changed in the unstable build, and the change is coming up in the next update. :)

> Also, many players have difficulty restoring the wires leading to the docking port of a beacon.

I might be missing something, but why would you need to restore the wiring to the docking port? It should be entirely possible to get the station back in action even if the docking port non-functional.

**Reply by pavel4win** [reply]
I agree, restoring the beacon's docking port is not necessary to restore its functionality.
Forgive my perfectionism.
